document-generator-api/src/test/java/uk/gov/companieshouse/document/generator/api/document/RetrieveApiEnumerationDescriptionTest.java

```diff
@@ -22,7 +22,7 @@
 
     private static final String DESCRIPTION_IDENTIFIERS_KEY_INVALID = "desc";
 
-    private static final String POPULATED_RESPONSE = "Abridged accounts made up to 01 October 2018";
```

Comment:
Is `POPULATED_RESPONSE` a good description here, given that the string contains a variable placeholder?